src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurEventNotification.java

```diff
@@ -0,0 +1,99 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import java.sql.Date;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+public class ConcurEventNotification extends PersistableBusinessObjectBase{   
+    protected int concurEventNotificationId;
```

Comment:
I don't think the serialVersionUID is in fact needed in this case, it was intentionally removed.